### ISSUE TITLE
More route validation

### ISSUE THREAD

**Issue by fancycade** [novaframework/nova#110]
This is a quality of life improvement that I wanted to take note of before I forgot.

Today I ran into an issue where the routes file I made compiled fine but led to 404 errors.

I had this:
```
{"/update/did/subject", {did_repo_admin_controller, update_did_subject, #{methods => [post]}}}
```
But I needed this:
```
{"/update/did/subject", {did_repo_admin_controller, update_did_subject}, #{methods => [post]}}
```
Obviously, this is user error. However, it is such a subtle difference that it would be helpful if the tooling could identify these types of issues too.

**Comment by burbas** [comment]
You are totally right. We are currently working with a new routing-module that will give you warnings on such errors. So hopefully we can resolve this soon.

**Comment by fancycade** [comment]
Great! I'm very excited about the ongoing routing work.

**Comment by burbas** [comment]
Closed with 0df89e678787f41cb8421ec2d35cd8cfb1cb8286
